src/main/java/seedu/address/model/person/UniquePersonList.java

```diff
@@ -103,6 +104,13 @@ public void setPersons(List<? extends ReadOnlyPerson> persons) throws DuplicateP
         return FXCollections.unmodifiableObservableList(mappedList);
     }
 
+    /**
+     * Sorts person name in alphabetical order
+     */
+    public void sort() {
+        Collections.sort(internalList, Person.sort);
```

Comment:
try using lambda sort expression so that there's no need to write the comparator method, try: 
        `internalList.sort(Comparator.comparing(n -> n.getName().fullName));`